File: aat-lib/src/main/java/ch/bailu/aat_lib/gpx/interfaces/GpxBigDeltaInterface.java
package ch.bailu.aat_lib.gpx.interfaces;


import ch.bailu.aat_lib.gpx.attributes.GpxAttributes;

public interface GpxBigDeltaInterface extends GpxDeltaInterface  {


    long getPause();
    long getStartTime();
    long getEndTime();

    GpxType getType();

    GpxAttributes getAttributes();
}

File: aat-lib/src/main/java/ch/bailu/aat_lib/gpx/interfaces/GpxBigDeltaInterface.kt
package ch.bailu.aat_lib.gpx.interfaces

import ch.bailu.aat_lib.gpx.attributes.GpxAttributes

interface GpxBigDeltaInterface : GpxDeltaInterface {
    fun getPause(): Long
    fun getStartTime(): Long
    fun getEndTime(): Long
    fun getType(): GpxType
    fun getAttributes(): GpxAttributes
}
